src/hooks/index.js

```diff
@@ -175,48 +221,57 @@ exports.removeResponse = (excludeOptions) => (context) => {
 // non hook releated function
 exports.hasPermissionNoHook = (context, userId, permissionName) => {
 	const service = context.app.service('/users/');
-	return service.get({ _id: (userId || '') })
-		.then((user) => {
-			user.permissions = Array.from(user.permissions);
-			return (user.permissions || []).includes(permissionName);
-		});
+	return service.get({ _id: userId || '' }).then((user) => {
+		user.permissions = Array.from(user.permissions);
+		return (user.permissions || []).includes(permissionName);
+	});
 };
 
 exports.hasRoleNoHook = (context, userId, roleName, account = false) => {
 	const userService = context.app.service('/users/');
 	const accountService = context.app.service('/accounts/');
 	if (account) {
-		return accountService.get(userId)
-			.then((_account) => userService.find({ query: { _id: (_account.userId || ''), $populate: 'roles' } })
+		return accountService.get(userId).then((_account) =>
+			userService
+				.find({
+					query: { _id: _account.userId || '', $populate: 'roles' },
+				})
 				.then((user) => {
 					user.data[0].roles = Array.from(user.data[0].roles);
 
-					return (user.data[0].roles.filter((u) => (u.name === roleName)).length > 0);
-				}));
+					return (
+						user.data[0].roles.filter((u) => u.name === roleName)
+							.length > 0
+					);
+				}),
+		);
 	}
-	return userService.find({ query: { _id: (userId || ''), $populate: 'roles' } })
+	return userService
+		.find({ query: { _id: userId || '', $populate: 'roles' } })
 		.then((user) => {
 			user.data[0].roles = Array.from(user.data[0].roles);
 
-			return (user.data[0].roles.filter((u) => (u.name === roleName)).length > 0);
+			return (
+				user.data[0].roles.filter((u) => u.name === roleName).length > 0
+			);
 		});
 };
 
 const resolveToId = (service, key, value) => {
 	const query = {};
 	query[key] = value;
-	return service.find({ query })
-		.then((results) => {
-			const result = results.data[0];
-			if (!result) throw new TypeError(`No records found where ${key} is ${value}.`);
-			return result._id;
-		});
+	return service.find({ query }).then((results) => {
+		const result = results.data[0];
+		if (!result)
+			throw new TypeError(`No records found where ${key} is ${value}.`);
```

Comment:
Codacy found an issue: [Expected no linebreak before this statement.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)